base/server/python/pki/server/cli/config.py

```diff
@@ -0,0 +1,324 @@
+# Authors:
+#     Endi S. Dewata <edewata@redhat.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright (C) 2018 Red Hat, Inc.
+# All rights reserved.
+#
+
+from __future__ import absolute_import, print_function
+
+import getopt
```

Comment:
(To provide a little more feedback on this... The issue with `getopt` is that you lose the functionality of the script for the weeds of parsing).

Reply:
I was copying the code from an existing script. Let me take a look at argparse...

Reply:
Ah in which case it is fine; at some point it would be nice to update everything to `argparse` in master:

https://docs.python.org/3/library/getopt.html
https://docs.python.org/3/library/argparse.html

Reply:
@edewata I'll take this as an AI to update this to `argparse` afterwards.

Reply:
I created a ticket for that:
https://pagure.io/dogtagpki/issue/3072